gui/utils/utils.js

```diff
@@ -72,6 +72,37 @@ export const downloadFile = (fileId) => {
   }
 };
 
+
+export const deleteFile = (fileId) => {
+  const authToken = localStorage.getItem('accessToken');
+  const url = `${baseUrl()}/resources/delete/${fileId}`;
```

Comment:
This requires backend api changes as well in controllers/resouces.py. Can you add it in the PR.